src/components/events/event-card.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react"
+import Image from "next/image"
+import { Event } from "@/types/database-types"
+import AttendanceCard from "@/components/events/attendance-card"
+import { ClockIcon, MapPinIcon } from "@heroicons/react/24/outline"
+import {  CalendarIcon } from "@heroicons/react/24/solid"
+import {DayMapper, MonthMapper} from "@/enums/event.enum";
+import {getEventImage} from "@/helpers/event.helper";
+
+interface EventCardProps {
+  event: Event
+}
+const EventCard = ({ event }: EventCardProps) => {
+  const getDayAndMonth = (timestamp: string) => {
+    const date = new Date(timestamp)
+    return `${DayMapper[date.getDay()]}, ${MonthMapper[date.getMonth()]} ${date.getDate()}`
+  }
+
+  const getStartAndEndTime = (startTimeStamp: string, endTimeStamp: string | null) => {
+    let endDate
+    const startDate = new Date(startTimeStamp)
+    const startDateTime = startDate.getHours() + ":" + startDate.getMinutes()
+
+    if (endTimeStamp != null) {
+      endDate = new Date(endTimeStamp)
+      const endDateTime = endDate.getHours() + ":" + endDate.getMinutes()
+      return `${startDateTime} - ${endDateTime}`
+    }
+
+    return startDateTime
+  }
```

Comment:
You can do this easily with format of date-fns
This renders Wednesday, June 14 for example:

https://github.com/ASVGay/the-rhapsodies/blob/4d0ee3b13a48edab26fc15a5fda3a4494fab1ef3/src/components/events/event-info-card.tsx#L28 

You can also check that file to see how I did it with the time. See the [docs of date-fns](https://date-fns.org/v2.30.0/docs/format) for more examples